tests/unit/test_utils.py

```
@@ -802,6 +802,15 @@ def test_exclude_patterns_removes_matching_files(self):
                     sorted(returned_file_list), sorted(expected_files)
                 )
 
+    def test_file_not_found_error_raised_on_invalid_directory_provided(self):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_file_not_found_error_raised_on_invalid_directory_provided`

Add return type annotation: `None`

(ANN201)